### DISCUSSION TITLE
Issue with Installing @reduxjs/toolkit in Next.js 15

### DISCUSSION THREAD

**Discussion by jsdev-robin** [vercel/next.js#72820] [Help]
I'm encountering an issue while trying to install @reduxjs/toolkit in a Next.js 15 project. After running npm install @reduxjs/toolkit, I'm facing compatibility issues, possibly due to dependencies or changes in Next.js 15 that affect Redux toolkit integration.

I would appreciate any insights on:

Whether Next.js 15 has introduced breaking changes that affect Redux or Redux Toolkit.
Suggested solutions or workarounds for integrating Redux Toolkit with Next.js 15.
Looking forward to hearing your thoughts and any advice on how to resolve this issue. Thanks!

### Additional information

```js
Additional Information:

Next.js version: 15.0.3
Redux Toolkit version: Latest stable version (@reduxjs/toolkit).

Next.js 15 has introduced breaking changes that affect Redux or Redux Toolkit.
Suggested solutions or workarounds for integrating Redux Toolkit with Next.js 15.
If others have encountered similar issues and any troubleshooting steps they have found effective.
```

**Comment by icyJoseph** [comment]
Hi,

Next 15 uses React 19. 

- https://github.com/reduxjs/redux-toolkit/issues/4540
